flink-connector-aws-base/pom.xml

```diff
@@ -52,6 +52,11 @@ under the License.
             <artifactId>netty-nio-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>apache-client</artifactId>
+        </dependency>
```

Comment:
We will need to ensure this is shaded correctly and NOTICE files are updated

Reply:
Verified that apache-client is in the installed JAR.

<img width="461" alt="Screenshot 2022-12-16 at 19 51 45" src="https://user-images.githubusercontent.com/108749182/208178095-b0490c6f-67c7-4b59-90e2-811415ad2004.png">


Verified that NOTICE files are updated.

```

- software.amazon.awssdk:apache-client:2.17.247
- software.amazon.awssdk:netty-nio-client:2.17.247
```

Reply:
NOTICE files do not update automatically, so either:
- It is already included and this is not necessary, or
- The NOTICE files need updating

Since flink-connector-aws-base is common to all connectors we will need to update all the NOTICE files I believe.

Reply:
Yes, I already see it. For all connectors, thanks